src/main/java/build/buildfarm/common/redis/BalancedRedisQueue.java

```diff
@@ -46,8 +46,9 @@
  *     the same underlying redis queues.
  */
 public class BalancedRedisQueue {
-  private static final Duration START_TIMEOUT = Duration.ofSeconds(1);
+  public static final int UNLIMITED_QUEUE_DEPTH = -1;
```

Comment:
Consolidate this value from the config - they shouldn't have different representations